package/README.md

```diff
@@ -1,44 +1,73 @@
 
-## Requirements
+### 1. Install with npm (recommended)
 
-### Build tool configuration
+We recommend [installing GOV.UK Frontend using node package manager
+(npm)](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md).
 
-When compiling the Sass files you'll need to define includePaths to reference the node_modules directory. Below is a sample configuration using gulp
+### 2. Install by using compiled files
 
-    .pipe(sass({
-      includePaths: 'node_modules/'
-    }))
+You can also [download the compiled and minified assets (CSS, JavaScript) from
+GitHub](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-from-dist.md).
 
-### Static asset path configuration
+## Importing styles
 
-To show the button image you need to configure your app to show these assets. Below is a sample configuration using Express js:
+You need to import the GOV.UK Frontend styles into the main Sass file in your
+project. You should place the below code before your own Sass rules (or Sass
+imports) if you want to override GOV.UK Frontend with your own styles.
 
-    app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/assets')))
+To import add the below to your Sass file:
 
-### Setting up Nunjucks views and paths
+  ```scss
+  @import "node_modules/govuk-frontend/all";
+  ```
 
-Below is an example setup using express configure views:
+[More details on importing styles](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#importing-styles)
+
+## Importing JavaScript
+
+Some of the JavaScript included in GOV.UK Frontend improves the usability and
+accessibility of the components. You should make sure that you are importing and
+initialising Javascript in your application to ensure that all users can use it successfully.
+
+You can include Javascript for all components either by copying the `all.js` from `node_modules/govuk-frontend` into your application or referencing the file directly:
+
+```html
+<script src="<path-to-govuk-frontend-all-file>/all.js"></script>
+```
+Next you need to initialise the script by adding:
+
+```html
+<script>window.GOVUKFrontend.initAll()</script>
+```
+
+[More details on importing Javascript and advanced options](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#importing-javascript)
+
+
+## Importing assets
+
+In order to import GOV.UK Frontend images and fonts to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
+
+[More details on importing assets](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#import-assets)
 
-    nunjucks.configure('node_modules/govuk-frontend/components/', {
-      autoescape: true,
-      cache: false,
-      express: app
-    })
 
 ## Getting updates
```

Comment:
Can we update this text too?
_To check whether you have the latest version of the button run:_

Reply:
Good spot!